rust/crd/src/lib.rs

```diff
@@ -15,7 +15,7 @@ use stackable_operator::{
     config::{fragment, fragment::Fragment, fragment::ValidationError, merge::Merge},
     k8s_openapi::apimachinery::pkg::api::resource::Quantity,
     kube::CustomResource,
-    product_config_utils::{ConfigError, Configuration},
+    product_config_utils::Configuration,
```

Comment:
Optional: You could do something like this and not change the error types further down
```
    product_config_utils::{Configuration, Error as ConfigError},
```

Reply:
I see what you mean but I would prefer to not use the `ConfigError` name anymore.